src/main/java/uk/gov/pay/ledger/transaction/search/common/TransactionSearchParams.java

```diff
@@ -506,9 +506,8 @@ public String buildQueryParamString(Long forPage) {
         }
         queries.add("page=" + forPage);
         queries.add("display_size=" + getDisplaySize());
-
-
-        return String.join("&", queries);
+        
+        return encodeParam(String.join("&", queries));
```

Comment:
This seems a little convoluted… the same thing could be achieved  (at the cost) of modifying more lines by doing:

```java
if (isNotBlank(toSettledDate)) {
    queries.add(TO_SETTLED_DATE_FIELD + "=" + URLEncoder.encode(toSettledDate, UTF-8));
}
if (isNotBlank(metadataValue)) {
    queries.add(METADATA_VALUE + "=" + URLEncoder.encode(metadataValue, UTF-8));
}
```

etc.